### PR TITLE
feat: Implement Testimonials Section on Homepage #59

### PR DESCRIPTION
**Pull request by nishant0708**
feat: Implement Testimonials Section on Homepage #59

#59

-Added testimonial section in homepage
-Responsive as well

https://github.com/Tanay-ErrorCode/lupo-skill/assets/101548649/412b708d-ceff-4d78-9179-467151a6b5bc

### CODE REVIEW

File: src/components/testimonial/testimonial.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  text-align: center;
+  color: white;
+  margin: 50px 0px;
+}
+.carousel-item {
+  padding: 20px 0;
+}
+
+.carousel-indicators {
+  transform: translateY(4rem);
```
Comment (Doraemon012) [inline src/components/testimonial/testimonial.css:11]: Increase little bit gap between the indicators.

File: src/components/testimonial/TestimonialCard.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      display: 'flex',
+      flexDirection: 'column',
+      borderRadius: "15px",
+      margin: isSmallScreen ? '0 auto' : 'initial'
+    }}>
+      <CardContent sx={{ flexGrow: 1 }}>
+        <Box display="flex" flexDirection="column" alignItems="center" justifyContent="space-between" height="100%">
+          <Box display="flex" alignItems="center" flexWrap="wrap" justifyContent="center" gap="20px">
+            <Avatar alt={name} src={image} sx={{ width: 56, height: 56, border: "1.5px solid #AE8892" }} />
+            <Typography variant="h5" fontSize={isSmallScreen ? "20px" : "24px"} fontWeight="700" whiteSpace="nowrap">
```
Comment (Doraemon012) [inline src/components/testimonial/TestimonialCard.jsx:29]: Do not give the font size in `px`, instead use `rem`.

File: src/components/HomePage/HomePage.tsx [hunk 3]
```diff
@@ -278,6 +283,11 @@ const HomePage = () => {
                 </div>
               </Grid>
             </Grid>
+            {/* testimonial */}
+            <Box>
+              <h1 className="happy">Happy Users</h1>
```
Comment (Doraemon012) [inline src/components/HomePage/HomePage.tsx:288]: Make this bold.

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Please make the requested changes

**Comment by Doraemon012** [comment]
@nishant0708
1. On **tablet screens, 2 cards** should be visible, **1 on phone**.
2. The component should be defined in a `tsx` file instead of `jsx`
3. Remove the added image
4. Use this sample data:
``` tsx
const testimonials = [
  {
    name: 'Elena Rodriguez',
    text: "I can't express enough how much I appreciate this platform. The live events are incredibly engaging, and the articles provide valuable insights. I've learned so much, and it's all thanks to this website!",
    image: 'https://randomuser.me/api/portraits/women/9.jpg',
  },
  {
    name: 'Michael Chang',
    text: "As a busy professional, I struggle to find time for skill development. This website has been a lifesaver! The resources are concise yet comprehensive, and the live events fit perfectly into my schedule. Highly recommended for anyone looking to grow professionally.",
    image: 'https://randomuser.me/api/portraits/men/2.jpg',
  },
  {
    name: 'Sophia Patel',
    text: "I stumbled upon this website while searching for ways to enhance my creative skills. It's been a revelation! The variety of topics covered is impressive, and I've found the live events to be both informative and inspiring. Thank you for creating such a valuable resource!",
    image: 'https://randomuser.me/api/portraits/women/3.jpg',
  },
  {
    name: 'David Thompson',
    text: "I've been a part of many skill-sharing platforms, but none compare to this one. The community here is supportive, the content is top-notch, and the live events feel like attending a masterclass. Kudos to the team behind this fantastic initiative!",
    image: 'https://randomuser.me/api/portraits/men/4.jpg',
  },
  {
    name: 'Emma Johnson',
    text: "This website has reignited my passion for learning! The articles are well-written and insightful, and the live events provide a unique opportunity to interact with industry experts. I've already recommended it to all my friends!",
    image: 'https://randomuser.me/api/portraits/women/5.jpg',
  },
  {
    name: 'Jonathan Lee',
    text: "I've always believed in the power of continuous learning, and this website has made it incredibly easy for me to pursue that belief. From workshops to webinars, there's something for everyone here. It's become my go-to platform for skill development!",
    image: 'https://randomuser.me/api/portraits/men/10.jpg',
  },
  {
    name: 'Maria Sanchez',
    text: "I'm amazed by the quality of content available on this platform. Whether you're a beginner or an expert, there's always something new to learn. The live events are particularly impressive, with knowledgeable speakers sharing valuable insights. I couldn't be happier with my experience!",
    image: 'https://randomuser.me/api/portraits/women/7.jpg',
  },
  {
    name: 'Chris Wilson',
    text: "I've been using this website for a few months now, and it's exceeded all my expectations. The resources are comprehensive, the articles are well-researched, and the live events are nothing short of brilliant. It's like having a personal mentor at your fingertips!",
    image: 'https://randomuser.me/api/portraits/men/8.jpg',
  },
];
```

**Comment by nishant0708** [comment]
Hey in small screen then will I have to increase number of indicator??

**Comment by Doraemon012** [comment]
> Hey in small screen then will I have to increase number of indicator??

@nishant0708 Yes

**Comment by nishant0708** [comment]
@Doraemon012  please check these changes ..and tell me if any changes require

**Comment by Tanay-ErrorCode** [comment]
@nishant0708 Please add sliding animation to the carousel.

**Comment by nishant0708** [comment]
@Tanay-ErrorCode @Doraemon012  please tell me changes required??

**Comment by Doraemon012** [comment]
@nishant0708 

Please increase the margin between two cards (on small devices)

image

</br>
The height of all cards should be same. Also, remove the inconsistency in margins between cards.

image

**Comment by nishant0708** [comment]
hey @Doraemon012 I tried but when i try to maintain same height of all the card the content is overflowing and when i try to fix overflowing issure the card heights get different so can you tell me or give a idea what can i do for that

**Comment by nishant0708** [comment]
Hey @Doraemon012  @Tanay-ErrorCode  please check this Latest commit. It Will Work Fine Now

**Review by Doraemon012 — APPROVED** [review]
@nishant0708 Great work !

**Comment by Tanay-ErrorCode** [comment]
@nishant0708 Good work!

**Pull request merged**